changelog.md

```diff
@@ -99,7 +101,7 @@
   use the Nim VM in a native Nim application.
 - Added the parameter ``val`` for the ``CritBitTree[T].incl`` proc.
 - The proc ``tgamma`` was renamed to ``gamma``. ``tgamma`` is deprecated.
-
+- The procs ``parseHexInt`` and ``parseOctInt`` now fail on empty strings and strings containing only valid prefixes, e.g. "0x" for hex integers.
```

Comment:
This should be under "breaking changes to the stdlib"

Reply:
Ok, I wasn't sure!